tests/android/androidTestWithCustomPort.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { baseTest } from '../config/baseTest';
+import { PageTestFixtures, PageWorkerFixtures } from '../page/pageTestApi';
+import type { AndroidDevice, BrowserContext } from 'playwright-core';
+export { expect } from '@playwright/test';
+
+type AndroidWorkerFixtures = PageWorkerFixtures & {
+  androidDevice: AndroidDevice;
+  androidContext: BrowserContext;
+};
+
+export const androidTest = baseTest.extend<
```

Comment:
This file is essentially the same as `androidTest.ts` and contains a lot of duplicate logic.

Reply:
Reused androidTest.ts file